test/e2e/e2e_test.go

```diff
@@ -166,8 +166,8 @@ var _ = Describe("controller", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred(), "Failed to parse pod JSON")
 
 			finalizers, found := podObj["metadata"].(map[string]interface{})["finalizers"].([]interface{})
-			Expect(found).To(BeTrue(), "Pod does not have finalizers")
-			Expect(finalizers).To(ContainElement("org.instaslice/accelerator"), "Finalizer org.instaslice/accelerator not found on pod")
+			finalizer := orgInstaslicePrefix + "accelerator"
```

Comment:
This constant `orgInstaslicePrefix` has to be defined inside this `test/e2e` package